### DISCUSSION TITLE
Issue with OptimizedApp check

### DISCUSSION THREAD

**Discussion by rcerljenko** [spatie/laravel-health#110] [General]
Exception:

```
The check named OptimizedApp did not complete. An exception was thrown with this message: BadMethodCallException: There's no value ok defined for enum Spatie\Health\Enums\Status, consider adding it in the docblock definition.
```

**Comment by rcerljenko** [comment]
figured it out! your Enum package depends on phpdoc via reflection
this is all great in dev env when opcache is not enabled but in production opcache strips all comments from the code unless you instruct it not do do so via php.ini directive => `opcache.save_comments=1`

took me a while to figure it out tho...

**Reply by Jul000** [reply]
Thanks for the solution

**Reply by clementmas** [reply]
That's more of a workaround. I'm not sure I'll use it. Thanks for sharing it though.
